src/intl/en/page-eth.json

```diff
@@ -15,11 +15,13 @@
   "page-eth-flexible-amounts": "Available in flexible amounts",
   "page-eth-flexible-amounts-desc": "ETH is divisible up to 18 decimal places so you don't have to buy 1 whole ETH. You can buy fractions at a time – as little as 0.000000000000000001 ETH if you want.",
   "page-eth-fuels": "ETH fuels and secures Ethereum",
-  "page-eth-fuels-desc": "ETH is the lifeblood of Ethereum. When you send ETH or use an Ethereum application, you'll pay a small fee in ETH to use the Ethereum network. This fee is an incentive for a miner to process and verify what you're trying to do.",
-  "page-eth-fuels-desc-2": "Miners are like the record-keepers of Ethereum – they check and prove that no one is cheating. Miners who do this work are also rewarded with small amounts of newly-issued ETH.",
+"page-eth-fuels-desc": "ETH is the lifeblood of Ethereum. When you send ETH or use an Ethereum application, you'll pay a fee in ETH to use the Ethereum network. Part of the fee is required by the network (<a to=\"/glossary/#base-fee\">base fee</a>), and part is an incentive for a block producer (priority fee) to process and include what you're trying to do.",
+  "page-eth-fuels-desc-2": "Currently, miners are like the record-keepers of Ethereum—they check and prove that no one is cheating, and perform work for the right to propose a block of transactions. Miners who do this work are also rewarded with small amounts of newly-issued ETH.",
   "page-eth-fuels-desc-3": "The work miners do keeps Ethereum secure and free of centralized control. In other words,",
+  "page-eth-fuels-desc-4": "Anyone is free to run Ethereum client software to verify and hold the state of the chain themselves. Node operators are not required to perform the same work as miners, and can quickly verify blocks that are proposed by others to ensure their validity, and to ensure they always have permissionless and private access to the network.",
```

Comment:
I like the idea of introducing nodes early, but IMO this is far too complex to introduce here. 

These are the things we assume users understand in this section:

- **Ethereum client software** 
- **verify and hold the state of the chain**
- **Node operators** 
- **verify blocks**
- **are proposed**
- **to ensure their validity**